Frameworks/AEPSwiftUI/Sources/ContentCards/ContentCardCustomizer.swift

```diff
@@ -0,0 +1,19 @@
+/*
+ Copyright 2024 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+/// Protocol for customizing Content Card templates
+public protocol ContentCardCustomizer {
+    /// Implement this function to customize content cards with SmallImageTemplate
+    func customize(template: SmallImageTemplate)
```

Comment:
will we eventually have multiple `customize` methods in this protocol?  would it be better to just pass in `ContentCardTemplate` as the param to cover for all future templates?